__tests__/app.test.js

```diff
@@ -57,4 +57,44 @@ describe("2. GET /api/reviews", () => {
        });
       });
   });
+});
+
+describe("7. POST /api/reviews/:review_id/comments", () => {
+  test("status: 200, returns object with posted comment ", () => {
+    const comment = { username: "dav3rid", body: "could be better" };
+    return request(app)
+      .post("/api/reviews/3/comments")
+      .send(comment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
+          comment_id: 7,
+          votes: 0,
+          created_at: expect.any(String),
+          author: comment.username,
+          body: comment.body,
+          review_id: 3,
+        });
+      });
+  });
+  test("status: 404, msg: resource not found - invalid review_id", () => {
```

Comment:
This test is good but is actually testing for a **valid** but non-existant review ID. 1000 is a valid id but it just doesn't exist.